auth/team.go

```diff
@@ -11,18 +11,21 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/tsuru/tsuru/db"
-	"github.com/tsuru/tsuru/log"
-	"github.com/tsuru/tsuru/permission"
-	"gopkg.in/mgo.v2"
-	"gopkg.in/mgo.v2/bson"
+        "github.com/tsuru/tsuru/log"
+        "github.com/tsuru/tsuru/permission"
+        "github.com/tsuru/tsuru/validation"
+        "gopkg.in/mgo.v2"
+        "gopkg.in/mgo.v2/bson"
 )
 
 var (
-	ErrInvalidTeamName   = errors.New("invalid team name")
+	ErrInvalidTeamName   = errors.New(
```

Comment:
could you also wrap this message on an "ValidationError" from the tsuru error package?